<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KbSimpleOptionsWidget</class>
 <widget class="QWidget" name="KbSimpleOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Keyboard layout switch keys</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="klsAltShift">
        <property name="text">
         <string>Alt + Shift</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="klsCtrlShift">
        <property name="text">
         <string>Ctrl + Shift</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="klsAltSpace">
        <property name="text">
         <string>Alt + Space</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Other options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="ktermXorg">
        <property name="text">
         <string>Ctrl+Alt+Backspace terminates xorg</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
